hive_agent/llm_settings.py

```diff
@@ -1,16 +1,18 @@
 import os
 import logging
+from dotenv import load_dotenv
+load_dotenv()
+from langtrace_python_sdk import langtrace
+langtrace.init(api_key = os.getenv('LANGTRACE_API_KEY'))
```

Comment:
if the user doesn't set this in their environment, then don't import it or init it. if they do, then init it